spyder_kernels/customize/spydercustomize.py

```diff
@@ -347,10 +347,17 @@ def set_spyder_breakpoints(self, breakpoints):
             # Do 'continue' if the first breakpoint is *not* placed
             # where the debugger is going to land.
             # Fixes issue 4681
-            if breaks and lineno != breaks[0] and osp.isfile(fname):
-                get_ipython().kernel.pdb_continue()
+            if self.continue_if_has_breakpoints and \
+                    breaks and \
+                    lineno < breaks[0] and \
+                    osp.isfile(fname):
```

Comment:
Please use parenthesis instead of line continuations here, like this:

```python
if (self.continue_if_has_breakpoints and
        breaks and
        lineno < breaks[0] and
        osp.isfile(fname)):
    ...
```